kc-cloudtrust-module/pom.xml

```diff
@@ -22,6 +22,7 @@
     <properties>
         <cloudtrust.version>3.0.0</cloudtrust.version>
         <keycloak.version>18.0.0</keycloak.version>
+        <kc-cloudtrust.version>18.0.0</kc-cloudtrust.version>
```

Comment:
We should have here cloudtrust.version 3.0.0-SNAPSHOT and kc-cloudtrust.version 18.0.0-SNAPSHOT
When building a RELEASE version, definitions should be replaced by version numbers without -SNAPSHOT
Ideally, when building a RELEASE, you can grep SNAPSHOT: if any code line matches, make the build fail

Reply:
As said above : https://github.com/cloudtrust/cloudtrust-parent/pull/53#discussion_r1102507673 
For now, I was not able make things work this way